internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -1998,3 +1998,37 @@ func pauseOnFailure(callback func(runtime.Sequence, interface{}) (runtime.TaskEx
 		}, name
 	}
 }
+
+func canLeaveEtcd() bool {
+	etcdID := (&services.Etcd{}).ID(nil)
```

Comment:
I feel we should move the check into the task if possible; first of all, it makes it closer to the action in time, second, we might try to ignore failures (?)

we could e.g. check for the existence of etcd data directory for example

Reply:
Moved, added folder check (same as in CACPPT)